pkg/model/config/config.go

```diff
@@ -20,6 +20,7 @@ const (
 	// Scaffolding versions
 	Version1 = "1"
 	Version2 = "2"
+	Version3 = "3-alpha"
```

Comment:
Will the plugins be a breaking change for the layout for the scaffold files?
 I do not think so, in this way IHMO the V3 should not be added. The breaking change will be clear in the new bump tag version as 3.0.0. Note that V2 was created just because the structure of files scaffolded changed.